content/go/concepts/math-functions/terms/atanh/atanh.md

```diff
@@ -0,0 +1,138 @@
+---
+Title: 'Atanh()'
+Description: 'Returns the inverse hyperbolic tangent of the given value.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Trigonometry'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Atanh()`** function takes a float64 argument representing the angle whose inverse hyperbolic tangent value you want to find. It returns this inverse hyperbolic tangent as a float64 value that can range from negative infinity to positive infinity.
+
+# Syntax
+
+```pseudo
+result := math.Atanh(x)
+```
+
+# Example
+
+The following code calculates the arctangent and prints out the result:
+
+```go
+package main
+
+import "fmt"
+import "math"
+
+func main() {
+	
+	fmt.Println(math.Atanh(0.5))
+    fmt.Println(math.Atanh(math.NaN()))
+
+} 
+```
+
+The output will be
+```shell
+0.5493061443340548
+NaN
+```
+
+# Source file
```

Comment:
Remove this source file section. It isn't the kind of documentation we're doing in Docs.

Reply:
I have eliminated the source file section